### PR TITLE
feat: use local docs folder instead of git repo

### PR DESCRIPTION
**Pull request by PatrickHeneise**
If not git repo is present in the environment configuration, use a local folder (`DOCS_PATH`) instead.

### CODE REVIEW

**Review by MoSattler — APPROVED** [review]

File: src/components/CodeBlock.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { useState } from 'react'
 
 const copyIcon = (
   <svg
-    className="text-opstrace-600 h-6 w-6"
+    className="docs-codeblock-copy-icon"
```
Comment (C-Mejlak) [inline src/components/CodeBlock.jsx:9]: The tailwind classes will need to be applied with the `@apply`

File: src/components/CodeBlock.jsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const copyIcon = (
 )
 const doneIcon = (
   <svg
-    className="text-opstrace-600 h-6 w-6"
+    className="docs-codeblock-done-icon"
```
Comment (C-Mejlak) [inline src/components/CodeBlock.jsx:20]: same as above

File: src/components/CodeBlock.jsx [hunk 3]
```diff
@@ -48,9 +52,7 @@ export default function CodeBlock({ children, className = '', theme = github })
               text={children.trim()}
               onCopy={() => setCopied(doneIcon)}
             >
-              <button className="absolute top-3 -right-8 text-opstrace-600">
-                {copied}
-              </button>
+              <button className="docs-codeblock-btn">{copied}</button>
```
Comment (C-Mejlak) [inline src/components/CodeBlock.jsx:55]: same as above

File: src/components/Tabs.jsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ const MobileTabs = ({ className, options, value, onChange }) => (
       value={value}
       onChange={onChange}
       id="tabs"
-      className="block w-full pl-3 pr-10 py-2 text-base border-gray-300 focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm rounded-md"
+      className="docs-tabs-mobile-select"
```
Comment (C-Mejlak) [inline src/components/Tabs.jsx:13]: same as above

**Comment by PatrickHeneise** [comment]
This turned out to be a little more complicated, so I'll close this for now and prepare a new PR with a few more adjustments.

**Pull request closed without merging**